Tasks/kudaka/final project/project_sites/media_site/add_content/models.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+
+from django.db import models
+from django.urls import reverse
+
+
+# Create your models here.
+class Media_models(models.Model):
```

Comment:
the naming here is not very good, it's not a mistake, but there is a convention to call such a classes just like the object they represent, without a word 'model'